src/main/java/com/alkemy/ong/entity/SlideEntity.java

```diff
@@ -2,19 +2,13 @@
 
 import lombok.*;
 import org.hibernate.annotations.GenericGenerator;
-import org.hibernate.annotations.SQLDelete;
-import org.hibernate.annotations.Where;
-
 import javax.persistence.*;
 
 @Table(name = "slides")
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-@Getter
-@Setter
-@SQLDelete(sql = "UPDATE slides SET softDeleted = true WHERE id=?") //This query provides the soft delete, as an update over slides
-@Where(clause = "softDeleted=false")
+@Data
```

Comment:
Ana, why did you remove the softDelete from SlideEntity? any special reason?

Reply:
![Slide_entity](https://user-images.githubusercontent.com/50774097/156818010-6db11a69-3538-487e-a3c4-3746b236c2c8.png)
You are right! thanks for your comment, there was no attribute in the ticket but it should go, I will re added it

Reply:
Added the attribute and re added query and where clause, also updated main